src/services/Path.php

```diff
@@ -215,7 +216,9 @@ public function getConfigDeltaPath(bool $create = true): string
      */
     public function getRebrandPath(bool $create = true): string
     {
-        $path = $this->getStoragePath($create) . DIRECTORY_SEPARATOR . 'rebrand';
+        $path = App::env('CRAFT_REBRAND_PATH')
+            ? App::parseEnv('$CRAFT_REBRAND_PATH')
+            : $this->getStoragePath($create) . DIRECTORY_SEPARATOR . 'rebrand';
```

Comment:
Couldn’t this be simplified to just

```suggestion
        $path = App::env('CRAFT_REBRAND_PATH')
            ?: ($this->getStoragePath($create) . DIRECTORY_SEPARATOR . 'rebrand');
```

Or am I missing something here?

Reply:
I used `parseEnv` so it would resolve aliases, so that you could do:

`CRAFT_REBRAND_PATH=@root/storage/rebrand`

Otherwise, on Cloud, you'd have to know to use: `/var/task/storage/rebrand`